CHANGELOG.md

```diff
@@ -33,7 +33,6 @@ and the project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.
 - Adapt tutorials to the renaming from PR #9 ([#17](https://github.com/microsoft/syntheseus/pull/17)) ([@jagarridotorres])
 - Pin `pydantic` version to `1.*` ([#10](https://github.com/microsoft/syntheseus/pull/10)) ([@kmaziarz])
 - Fix compatibility with Python 3.7 ([#5](https://github.com/microsoft/syntheseus/pull/5)) ([@kmaziarz])
-- Pin `zipp` version to `<3.16` ([#11](https://github.com/microsoft/syntheseus/pull/11)) ([@kmaziarz])
```

Comment:
The CHANGELOG is supposed to record changes between consecutive versions, and #11 got effectively reverted, so I'm also dropping the corresponding entry.